specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2024-07-01/amlfilesystem.json

```diff
@@ -0,0 +1,2266 @@
+              "x-ms-mutability": [
+                "read",
+                "create"
+              ]
+            },
+            "maximumErrors": {
+              "description": "Total non-conflict oriented errors the import job will tolerate before exiting with failure. -1 means infinite. 0 means exit immediately and is the default.",
+              "type": "integer",
+              "format": "int32",
+              "default": 0,
+              "x-ms-mutability": [
+                "read",
+                "create"
+              ]
+            },
+            "status": {
+              "x-ms-client-flatten": true,
+              "description": "The status of the import",
+              "type": "object",
+              "readOnly": true,
+              "properties": {
+                "state": {
+                  "readOnly": true,
+                  "description": "The operational state of the import job. InProgress indicates the import is still running. Canceled indicates it has been canceled by the user. Completed indicates import finished, successfully importing all discovered blobs into the Lustre namespace. CompletedPartial indicates the import finished but some blobs either were found to be conflicting and could not be imported or other errors were encountered. Failed means the import was unable to complete due to a fatal error.",
+                  "type": "string",
+                  "enum": [
+                    "InProgress",
+                    "Cancelling",
+                    "Canceled",
+                    "Completed",
+                    "CompletedPartial",
+                    "Failed"
+                  ],
+                  "x-ms-enum": {
+                    "name": "ImportStatusType",
+                    "modelAsString": true
+                  }
+                },
+                "statusMessage": {
+                  "readOnly": true,
+                  "description": "The status message of the import job.",
+                  "type": "string"
+                },
+                "totalBlobsWalked": {
+                  "readOnly": true,
+                  "description": "The total blob objects walked.",
+                  "type": "integer",
+                  "format": "int64"
+                },
+                "blobsWalkedPerSecond": {
+                  "readOnly": true,
+                  "description": "A recent and frequently updated rate of blobs walked per second.",
+                  "type": "integer",
+                  "format": "int64"
+                },
+                "totalBlobsImported": {
+                  "readOnly": true,
+                  "description": "The total blobs that have been imported since import began.",
+                  "type": "integer",
+                  "format": "int64"
+                },
+                "importedFiles": {
+                  "readOnly": true,
+                  "description": "New or modified files that have been imported into the filesystem.",
```

Comment:
Similar as above. Please go through all similar properties to indicate it's the number of files.